src/controllers/message.controller.js

```diff
@@ -0,0 +1,25 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
+};
+
+export const postMessage = async (req, res) => {
+  const { text, author, roomId } = req.body;
+  const message = {
+    text,
+    time: new Date().toString(),
+    author,
+    roomId,
+  };
+
+  await Message.create(message);
+  emitter.emit('message', message);
```

Comment:
You are emitting an event with the 'message' string. It would be better to avoid hardcoding this string and instead store it in a constant. This way, if you decide to change the event name, you only have to change it in one place.